docs/data-warehouse/query-insights.md

```diff
@@ -57,7 +57,7 @@ There are three system views to provide answers to these questions:
   - Returns information about each completed SQL request/query.
  
 - [queryinsights.exec_sessions_history (Transact-SQL)](/sql/relational-databases/system-views/queryinsights-exec-sessions-history-transact-sql?view=fabric&preserve-view=true)
-  - Returns information about frequently run queries.
+  - Returns information about past sessions and executed queries.
```

Comment:
```suggestion
  - Returns information about completed sessions.
```

Reply:
@JaredWestover this is the correct description. Accept this suggestion as a co-authored commit, and we'll get this merged. Good catch. After the updated article is published, GitHub will list you as an article contributor.